lib/rubocop/cop/style/space_inside_string_interpolation.rb

```diff
@@ -19,25 +19,39 @@ class SpaceInsideStringInterpolation < Cop
         SPACE_MSG = 'Missing space around string interpolation detected.'.freeze
 
         def on_dstr(node)
+          each_style_violation(node) do |final_node, msg|
+            add_offense(final_node, :expression, msg)
+          end
+        end
+
+        private
+
+        def each_style_violation(node)
```

Comment:
Ah, I see your point. Guess this makes sense as well. I'd probably use the name `on_style_violation`, though. I think we can figure this out later...